example/plan_execution.cpp

```diff
@@ -85,7 +85,7 @@ int main(int argc, char* argv[])
 
         // Wait for the mode to be switched
         while (robot.getMode() != flexiv::MODE_PLAN_EXECUTION) {
-            std::this_thread::sleep_for(std::chrono::milliseconds(1));
+            std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
This can be too long for integrators, how about 100 ms?

Reply:
Agree, reverted